tools/ci_build/github/azure-pipelines/win-ci-fuzz-testing.yml

```diff
@@ -45,9 +45,8 @@ jobs:
      $Env:USE_MSVC_STATIC_RUNTIME=1
      $Env:ONNX_ML=1
      $Env:CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DProtobuf_USE_STATIC_LIBS=ON -DONNX_USE_LITE_PROTO=ON -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=$(buildArch)-windows-static"
-     python setup.py bdist_wheel
-     python -m pip uninstall -y onnx -qq
-     Get-ChildItem -Path dist/*.whl | foreach {pip --disable-pip-version-check install --upgrade $_.fullname}   
+     python -m pip --disable-pip-version-check uninstall -y onnx -qq
+     python -m pip --disable-pip-version-check install -r $(Build.SourcesDirectory)\tools\ci_build\github\linux\docker\scripts\manylinux\requirements.txt   
     workingDirectory: '$(Build.SourcesDirectory)\cmake\external\onnx'
     displayName: 'Install ONNX'
```

Comment:
TODO: move all "Install ONNX" steps to a template, and get protobuf from ORT's deps instead of vcpkg.